spec/requests/articles/articles_spec.rb

```diff
@@ -78,6 +78,18 @@
       end
     end
 
+    context "when query string is q=latest" do
+      let!(:featured_article) { create(:article, featured: true) }
+      let!(:not_featured_article) { create(:article, featured: false) }
+
+      before { get "/feed?q=latest" }
+
+      it "returns last published articles" do
+        expect(response.body).to include(featured_article.title)
+        expect(response.body).to include(not_featured_article.title)
```

Comment:
Thank you so much for adding this test! I have a couple of small suggestions for this test, one of which should make this a bit more intuitive for future contributors and another regarding one of the test's expectations.

Right now, line `89` checks that `response.body` includes `not_featured_article.title`, but I think what we actually want to check here is that `response.body` does not include `not_featured_article.title`, as shown below:
```suggestion
        expect(response.body).not_to include(not_featured_article.title)
```

Additionally, I think we could change the names here to make the test a bit more readable for future contributors by renamining `featured_article` and `not_featured_article` to something along the lines of:
```suggestion
        expect(response.body).to include(last_published_article.title)
        expect(response.body).to include(not_last_published_article.title)
```

Let me know if you have any questions about this! All-in-all, awesome job, @ccozkan . 😄

Reply:
Thanks for the feedback, I will update my PR accordingly :smile: I have a question though :thinking: , why we don't want 'not featured articles' to be a part of the feed?

Reply:
Hi, @ccozkan and my sincerest apologies for the delayed response -- I was out of the office! 😊 

Now that I've read through yours and @rhymes's comments, it seems I may have _also_ misunderstood the initial goal, and for that, I am so sorry! The goal of my feedback here was to make these tests a bit more intuitive: since we are adding the ability to access only the "latest" articles and not necessarily the featured articles, I think that it makes more sense to make the names in the spec more in-line with what we're trying to accomplish here. That being said, my thought was that `last_published_article` or even `latest_article` better described what we're looking for in the "GET /feed/latest" test and that `featured_article` wasn't a very descriptive name for what we are trying to test.

Regarding [this](https://github.com/forem/forem/pull/11956#discussion_r549022460) comment: I misunderstood the spec's intention and did not mean to pull you in the wrong direction!

Reply:
Ok I see now, no problem at all! It's been a good experience for me :relaxed:

Reply:
I'm so happy to hear that! 😊